_posts/2021-01-22-logarithmic-time-factoring-magic-trick.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: "Factoring in logarithmic time: a mathemagic trick"
+categories:
+author: Robin Jadoul
+meta: "Factoring in logarithmic time: a mathemagic trick"
+tags:
```

Comment:
tags: RSA Broken-Cryptography?

Reply:
Is there a difference between tags and categories? How should those be treated?

Reply:
I don't think Category is being used in any way yet. Jack has some ideas about how we could use it in future, but you could just put "News" here for now as a suitably generic category?